lldb/source/Plugins/SymbolLocator/Debuginfod/SymbolLocatorDebuginfodProperties.td

```diff
@@ -4,4 +4,14 @@ let Definition = "symbollocatordebuginfod" in {
   def ServerURLs : Property<"server_urls", "Array">,
     ElementType<"String">,
     Desc<"An ordered list of Debuginfod server URLs to query for symbols. This defaults to the contents of the DEBUGINFOD_URLS environment variable.">;
+  def SymbolCacheLocation: Property<"cache_location", "String">,
+    DefaultStringValue<"">,
+    Desc<"The location where symbol files should be cached. This defaults to LLDB's system cache location.">;
+  def EnableAutoLookup : Property<"lookup_mode", "Enum">,
+    DefaultEnumValue<"eLookupModeAlways">,
+    EnumValues<"OptionEnumValues(g_debuginfod_symbol_lookup_mode)">,
+    Desc<"Control when DEBUGINFOD servers are queried for symbols">;
+  def Timeout : Property<"timeout", "UInt64">,
+    DefaultUnsignedValue<0>,
+    Desc<"Timeout (in seconds) for requests made to a DEBUGINFOD server. A value of zero defaults to DEBUGINFOD_TIMEOUT environment variable (or 90 seconds).">;
```

Comment:
add a new bool property for async as mentioned above?

Reply:
symbols.enable-background-lookup already handles async.